src/main/java/name/velikodniy/vitaliy/fixedlength/formatters/Formatter.java

```diff
@@ -11,35 +8,36 @@
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
+import name.velikodniy.vitaliy.fixedlength.FixedLengthException;
+import name.velikodniy.vitaliy.fixedlength.annotation.FixedField;
 
-@SuppressWarnings("rawtypes")
 public abstract class Formatter<T> {
-    private static Map<Class<? extends Serializable>, Class<? extends Formatter>> defaultFormatters = new HashMap<>();
+    private static final Map<Class<? extends Serializable>, Class<? extends Formatter<?>>> DEFAULT_FORMATTERS = new HashMap<>();
```

Comment:
Now the line is 128 symbols. Code style violation was set to 120.
You can run `gradle checkstyleMain` task locally to check code.